packages/client/hmi-client/src/workflow/operator/tera-operator-outputs.vue

```diff
@@ -47,12 +47,19 @@ const emit = defineEmits([
 	'remove-edges'
 ]);
 
-defineProps({
+const props = defineProps({
 	outputs: {
-		type: Array as PropType<WorkflowPort[]>,
+		type: Array as PropType<WorkflowOutput<any>[]>,
 		default: () => []
 	}
 });
+
+const selectedOutputs = computed(() =>
+	props.outputs?.filter((output) => {
+		if (!('isSelected' in output)) return true;
+		return output.isSelected;
+	})
+);
```

Comment:
A way to expose output ports whilst making it backwards compatible with the way we are currently appending output ports from other nodes where the isSelected key isnt present